sdk/include/cheri.hh

```diff
@@ -812,6 +812,15 @@ namespace CHERI
 			return permission_set_from_pointer(ptr);
 		}
 
+		/**
+		 * Drop some permissions from this capability
```

Comment:
Is it worth extending this comment to say that you must use this if dropping global from a sealed cap and that it may produce very slightly smaller code?